README.md

```diff
@@ -75,9 +75,30 @@ Add the stylesheets by adding this line to your app/assets/stylesheets/applicati
 
 (Note: The Javascript includes require Blacklight and must be put after that.)
 
+## Updating to 4.0.0
+
+* [SimpleForm](https://github.com/plataformatec/simple_form) is supported from release 3.2.0 onwards
```

Comment:
Should we add a note about the breaking change in `#to_model`? It used to return the same value as `#model`, but now returns `self`.

Reply:
Apologies for that, a note for this has been added